docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1704,7 +1704,7 @@ How it **works**:
 Common **causes**:
 
 - Your code has a single target that exposes the same time series multiple times, or multiple targets with identical labels.
-- System time of your Prometheus instance has been shifted backwards. If this was a mistake, fix the system time back to normal. Otherwise, wait until the system time catches up to the time it was changed.
+- System time of your Prometheus instance has been shifted backwards. If this was a mistake, fix the system time back to normal. Otherwise, wait until the system time catches up to the time it was changed. Using [node exporter](https://github.com/prometheus/node_exporter) metrics identify clock skew on any hosts using this promQL query: `abs(node_time_seconds - timestamp(node_time_seconds))`.
```

Comment:
> abs(node_time_seconds - timestamp(node_time_seconds))

This query doesn't measure the clock skew of a target node. It's a rough measurement of the difference between the node timestamp and Prometheus timestamp when it received the scrape response, but you don't know which of the 2 nodes have a clock skew (maybe none, maybe one of the two, maybe both).

A better approach may be using timex metrics, like `node_timex_maxerror_seconds` and `node_timex_estimated_error_seconds`.

Reply:
Thanks for looking into this @pracucci. I have updated the PR. Could you please review it again and let me know if that looks good?